Sources/SpeziOnboarding/ConsentView.swift

```diff
@@ -197,6 +228,132 @@ public struct ConsentView<ContentView: View, Action: View>: View {
     }
 }
 
+
+/// Extension of ``ConsentView`` enabling the export of the signed consent page in the onboarding flow.
+extension ConsentView {
+    /// Creates a view representation of the consent content, ready for PDF export via SwiftUIs `ImageRenderer`.
+    /// At the moment, this is
+    ///
+    /// This function constructs a view for presenting the markdown consent form. It combines the
+    /// given markdown and the user's signature with details such as the date of export. It can be
+    /// used to create exportable PDF documents of the consent form.
+    ///
+    /// - Parameters:
+    ///   - markdown: The markdown consent content as an `AttributedString`.
+    ///
+    /// - Returns: A SwiftUI `View` representation of the consent content and signature.
+    ///
+    /// - Note: This function avoids the use of asynchronous operations.
+    /// Asynchronous tasks are incompatible with SwiftUI's `ImageRenderer`,
+    /// which expects all rendering processes to be synchronous.
+    func exportBody(markdown: AttributedString) -> some View {
+        VStack {
+            HStack {
+                Spacer()
+                
+                Text("Exported: \(DateFormatter.localizedString(from: Date(), dateStyle: .medium, timeStyle: .short))")
+                    .font(.caption)
+            }
+            .padding()
+            
+            Text("Spezi Consent")
+                .font(.title)
+            
+            Text(markdown)
+                .padding()
```

Comment:
Couldn't we already cover all use cases by just displaying the `contentView` here?

We should add some sort of logic that only enables the action button view once possibly async content in the `contentView` has loaded.

I am TBH also happy to drastically simplify the API signature of the Consent view if needed, e.g., making the markdown as well as the HTML non-async for now or removing the initializers and support for a custom content view and HTML if we only support MD for now. I suspect that this PR will be a breaking change anyways and we can always add this later again.

Reply:
Sure, that sounds good to me. The async functionality might still be usable if we do some good state management but I am fine with we drop the arbitrary supported views + HTML for now. As noted in the review: Feel free to modify the signature accordingly and to your needs, I am not too worried about breaking changes here 👍